locale/en/LC_MESSAGES/django.po

```diff
@@ -8,17 +8,59 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-10-10 08:20+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: Aurore Poirier <foss@borale.si>\n"
+"Report-Msgid-Bugs-To: contact@insalan.fr\n"
+"POT-Creation-Date: 2023-10-27 18:56+0200\n"
+"PO-Revision-Date: 2023-10-27 19:15+0200\n"
+"Last-Translator: Amélie Gonzalez <contact@vulpinecitrus.info>\n"
 "Language-Team: SysRez-Dev <contact@insalan.fr>\n"
 "Language: English\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+#: insalan/cms/apps.py:6
+msgid "Module de gestion de contenu"
+msgstr "Content Management Module"
+
+#: insalan/cms/models.py:19
+msgid ""
+"Des constantes non définies sont utilisées: {', '."
+"join(sorted(excess_constants))}"
+msgstr ""
+
+#: insalan/cms/models.py:27
+msgid "Nom du contenu"
+msgstr "Content name"
+
+#: insalan/cms/models.py:28 insalan/cms/models.py:32
+msgid "Contenu"
+msgstr "Content"
+
+#: insalan/cms/models.py:33
+msgid "Contenus"
+msgstr "Contents"
+
+#: insalan/cms/models.py:45
+msgid "Nom de la constante"
+msgstr "Constant name"
+
+#: insalan/cms/models.py:47
+msgid "Valeur de la constante"
+msgstr "Constant value"
+
+#: insalan/cms/models.py:50
+msgid "Constante"
+msgstr "Constant"
+
+#: insalan/cms/models.py:51
+msgid "Constantes"
+msgstr "Constants"
+
+#: insalan/cms/tests.py:26
+msgid "Des constantes non définies sont utilisées: inconnue, random"
```

Comment:
this is a hardcoded transaltion of variables names, is this correct?

Reply:
I thought about it and in my opinion if we are talking about variable names it should probably not be leaked/spewed out into an error destined to a user

Reply:
Still, I will probably just ammend my commit to not translate the variable names and leave that to be fixed later.